src/Qowaiv/Text/Regxx.cs

```diff
@@ -0,0 +1,23 @@
+﻿namespace Qowaiv.Text;
+
+internal static class Regxx
```

Comment:
The name Regxx could be improved. Since it is internal, it's not a major issue.

Reply:
I'm open for suggestions.